rasa_core/events/__init__.py

```diff
@@ -362,10 +362,14 @@ def as_story_string(self):
 
     @classmethod
     def _from_story_string(cls, parameters):
-        slot_key = first_key(parameters, default_key="name")
-
-        if slot_key:
-            return SlotSet(slot_key, parameters[slot_key])
+        slots = []
+        for slot_key, slot_val in parameters.items():
+            slots.append(SlotSet(slot_key, slot_val))
+
+        if len(slots) == 1:
```

Comment:
do we need this, can't we always return a list?

Reply:
we can, i added this, because i thought it was a problem in tests, but it was actually `name`. I'll change that